snuba/cli/bulk_load.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+import click
+
+from snuba import settings
+from snuba.datasets.factory import get_dataset
+from snuba.snapshots.postgres_snapshot import PostgresSnapshot
+
+
+@click.command()
+@click.option('--dataset', type=click.Choice(['events', 'groupedmessage', 'outcomes']),
+              help='The dataset to bulk load')
+@click.option('--source',
+              help='Source of the dump. Depending on the dataset it may have different meaning.')
+@click.option('--dest-table',
```

Comment:
I'd like to make this more abstract but I believe we need to see how we will use this in prod before deciding wether we can hardcode the temporary table name in the dataset like the other table names. Same if we decide to put it into config. I would address that once we tried this in prod.